odoo/addons/base/models/ir_sequence.py

```diff
@@ -356,15 +357,16 @@ def _alter_sequence(self, number_increment=None, number_next=None):
         for seq in self:
             _alter_sequence(self._cr, "ir_sequence_%03d_%03d" % (seq.sequence_id.id, seq.id), number_increment=number_increment, number_next=number_next)
 
-    @api.model
-    def create(self, values):
+    @api.model_create_multi
+    def create(self, vals_list):
         """ Create a sequence, in implementation == standard a fast gaps-allowed PostgreSQL sequence is used.
         """
-        seq = super(IrSequenceDateRange, self).create(values)
-        main_seq = seq.sequence_id
-        if main_seq.implementation == 'standard':
-            _create_sequence(self._cr, "ir_sequence_%03d_%03d" % (main_seq.id, seq.id), main_seq.number_increment, values.get('number_next_actual', 1))
-        return seq
+        seqs = super().create(vals_list)
+        for seq, vals in zip(seqs, vals_list):
+            main_seq = seq.sequence_id
+            if main_seq.implementation == 'standard':
+                _create_sequence(self._cr, "ir_sequence_%03d_%03d" % (main_seq.id, seq.id), main_seq.number_increment, vals.get('number_next_actual', 1))
```

Comment:
Same remark here: seems we could use main_seq.number_next_actual and remove usage of vals_list in the loop.